### PR TITLE
Styles update

### PR DESCRIPTION
**Pull request by amanpdyadav**
- Add translations for assistiveTexts
- Add assistive text for radio buttons
- Fix discount checkbox layout
- Update stepper theme to black
- Update footer theme to black
- Change buttons theme to black

Refs PV-144

### CODE REVIEW

File: src/common/shoppingCart/ShoppingCart.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ const ShoppingCart = ({
   address,
   prices,
   vehicleDetail,
-  parkingDurationType = ParkingDurationType.FIXED_PERIOD,
+  parkingDurationType = ParkingDurationType.OPEN_END,
```
Comment (mhieta) [inline src/common/shoppingCart/ShoppingCart.tsx:47]: I believe this type Enum name should be:
ParkingDurationType.**OPEN_ENDED**

**Review by mhieta — CHANGES_REQUESTED** [review]
One minor update. Looks good otherwise 👍

File: src/common/shoppingCart/ShoppingCart.tsx (hunk 1 above)
Comment (amanpdyadav) [reply src/common/shoppingCart/ShoppingCart.tsx]: Thanks for the review. Its updated.

**Review by mhieta — APPROVED** [review]
Good 💯

**Pull request merged**